Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+// define a tag according to the Docker tag rules https://docs.docker.com/engine/reference/commandline/tag/
```

Comment:
Is there typically a need for us to be able to read this type of config file, or if we were at the jenkins url could we view this config in a more readable form?